src/inmanta/server/services/databaseservice.py

```diff
@@ -124,22 +139,26 @@ def _add_gauge(self, name: str, the_gauge: CallbackGauge) -> None:
     def start_monitor(self) -> None:
         """Attach to monitoring system"""
 
+        # TODO not all these are correct / equivalent to previous asyncpg pool implementation
```

Comment:
Do we have a ticket for this?

Reply:
No, good idea, I'll create one and link it here!

Reply:
https://github.com/inmanta/inmanta-core/issues/8775